evidence/cli/command_cli.go

```diff
@@ -105,6 +105,11 @@ func evidenceDetailsByFlags(ctx *components.Context) (*coreConfig.ServerDetails,
 		return nil, errors.New("platform URL is mandatory for evidence commands")
 	}
 	platformToEvidenceUrls(serverDetails)
+
+	if serverDetails.GetUser() != "" && serverDetails.GetPassword() != "" {
```

Comment:
Could be even just `serverDetails.GetUser() != ""`, without a check for a password?

Reply:
@denys-jfrog if you try with web-login you get user with token not password.
we still need both validation.